aeron-client/src/main/java/io/aeron/logbuffer/Header.java

```diff
@@ -95,6 +96,16 @@ public int positionBitsToShift()
         return positionBitsToShift;
     }
 
+    /**
+     * Set the number of times to left shift the term count to multiply by term length.
+     *
+     * @param positionBitsToShift number of times to left shift the term count to multiply by term length.
+     */
+    public void positionBitsToShift(final int positionBitsToShift)
```

Comment:
These methods could be a fluent API.

Reply:
Should the existing methods `offset(int)` and `buffer(DirectBuffer)` also be changed for fluent API?

Reply:
Probably good for consistency.

Reply:
Done